tests/test_convert.py

```diff
@@ -548,6 +546,7 @@ def test_convert_s3_path_csv(
 
 
 def test_convert_s3_path_sqlite(
+    get_tempdir: str,
```

Comment:
`get_tempdir` is a bit confusing to me with a `str` type hint. `get_tempdir` seems like a function name. Is there a simpler name? what was wrong with `tmpdir`?

Reply:
Thanks for this comment - this could use some improvement and the naming was mostly of convenience at the time it was created. The name stems from a fixture which generates and afterwards destroys temporary directories for tests [found here](https://github.com/cytomining/CytoTable/blob/main/tests/conftest.py#L34). When the fixture is called this way through this test a function runs to yield a new temporary directory path string and destroys that temporary directory when the test completes (post yield). The design here is to help with testing isolation and parallel performance improvements I hope we can eventually reach (where each test should not share access to a single/central temporary directory).

I naively thought this might split the definition from that of what's found under native [`tempfile`](https://docs.python.org/3/library/tempfile.html) names. I'll be more explicit here by adding an `fx_` in an attempt to make this more distinguishable to this package's work while also calling out that a function runs to generate this string through a pytest fixture.